examples/simpletest.py

```diff
@@ -1,18 +1,18 @@
-from board import *
+from board import SCL, SDA
```

Comment:
Clarify what `simpletest` is and how it should be used.

Reply:
Done